src/includes/performance/create-transaction-bound-to-scope/apple.mdx

```diff
@@ -0,0 +1,57 @@
+## Create Transaction Bound to The Current Scope
+
+
+Our SDK can bind a transaction to the scope making it accessible to every method running within this scope by calling `Sentry#startTransaction` method with `bindToScope` parameter to `true`.
+
+In cases where you want to attach Spans to an already ongoing Transaction you can use `Sentry.span`. This method will return a `SpanProtocol` in case there is a running Transaction or a `Span` in case there is already a running Span, otherwise it returns `nil`.
```

Comment:
```suggestion
In cases where you want to attach Spans to an already ongoing Transaction you can use `SentrySDK.span`. This method will return a `SpanProtocol` in case there is a running Transaction or a `Span` in case there is already a running Span, otherwise it returns `nil`.
```